src/lib/import-export/import/importers/importer.ts

```diff
@@ -71,9 +71,45 @@ export abstract class BaseImporter extends EventEmitter implements Importer {
 			}
 		}
 
+		this.fixSiteUrl( siteId );
 		this.emit( ImportEvents.IMPORT_DATABASE_COMPLETE );
 	}
 
+	private async fixSiteUrl( siteId: string ) {
+		const server = SiteServer.get( siteId );
+		if ( ! server ) {
+			throw new Error( 'Site not found.' );
+		}
+
+		// Step 1: Fetch the current site URL
+		const { stdout: currentSiteUrl } = await server.executeWpCliCommand(
+			`option get siteurl --require=/tmp/sqlite-command/command.php`
+		);
+
+		if ( ! currentSiteUrl ) {
+			console.error( 'Failed to fetch site URL after import' );
+			return;
+		}
+
+		const studioUrl = `http://localhost:${ server.details.port }`;
+
+		const { stdout, stderr, exitCode } = await server.executeWpCliCommand(
```

Comment:
Seems `stdout` is not used anywhere, we could probably remove it from here.

Reply:
Updated! :smile: